appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
```

Comment:
`{build}` is a valid version number as far as appveyor knows. But I guess its not a good version number to use.